src/beacons/BeaconsContext.tsx

```diff
@@ -132,12 +135,15 @@ const BeaconsContextProvider: React.FC = ({children}) => {
   const deleteCollectedData = useCallback(async () => {
     if (!isBeaconsSupported) return;
     if (isInitializedRef.current) {
-      try {
-        await Kettle.deleteCollectedData();
-        return Promise.resolve();
-      } catch (error) {
-        return Promise.reject(error);
-      }
+      const deleteOK = await Kettle.deleteCollectedData()
+        .then(() => {
+          return true;
+        })
+        .catch((error) => {
+          Bugsnag.notify(error);
+          return false;
+        });
+      return deleteOK;
```

Comment:
`deleteOK` is being returned, but is never used where `deleteCollectedData` is used as far as I can see?
Maybe it can be simplified to something like
```
await Kettle.deleteCollectedData().catch((error) => {
  Bugsnag.notify(error);
});
```
(and then also update deleteCollectedData type definition and init value)

Reply:
Is not being used yes, we can simplify it a bit